demo-ui/app/uk/gov/ons/addressIndex/demoui/views/clericalTool.scala.html

```diff
@@ -154,182 +155,583 @@ <h3 class="green">@aresponse.total @messages("single.found")</h3>
+                                            <div class="taColumn4">
+                                            @address.nag.map{ nag => @nag.streetDescriptor}
+                                            </div>
+                                        </div>
+
                                         <div class="taRow">
                                             <div class="taColumn1">
-                                                @messages("clerical.subbuildingname")
+                                            @messages("clerical.token.locality")
                                             </div>
                                             <div class="taColumn2">
-                                                @aresponse.tokens.getOrElse("SubBuildingName", "")
+                                            @messages("clerical.locality")
                                             </div>
                                             <div class="taColumn3">
-                                                @address.nag.map{ nag => @nag.organisation }
+                                            @aresponse.tokens.getOrElse(Tokens.locality, "")
+                                            </div>
+                                            <div class="taColumn4">
+                                            @address.nag.map{ nag => @nag.locality}
                                             </div>
                                         </div>
+
                                         <div class="taRow">
                                             <div class="taColumn1">
-                                                @messages("clerical.buildingname")
+                                            @messages("clerical.token.town_name")
                                             </div>
                                             <div class="taColumn2">
-                                                @aresponse.tokens.getOrElse("BuildingName", "")
+                                            @messages("clerical.town_name")
                                             </div>
                                             <div class="taColumn3">
-                                                @address.nag.map{ nag => @nag.pao.paoText }
+                                            @aresponse.tokens.getOrElse(Tokens.townName, "")
+                                            </div>
+                                            <div class="taColumn4">
+                                            @address.nag.map{ nag => @nag.townName}
                                             </div>
                                         </div>
                                         <div class="taRow">
                                             <div class="taColumn1">
-                                                @messages("clerical.buildingnumber")
+                                            @messages("clerical.token.town_name")
```

Comment:
Leaving the duplicates token names for now is fine. I wouldn't be happy about converting the divs to a table as this overturns an agreement I had with Alan (there are many discussions on the internet on DIVs v tables for reactiveness etc.). It is not particularly hard to implement the equivalent of rowspan with divs and css

<div style="display: table; width:300px; border: 1px solid black; border-spacing: 2px;">
  <div style="display: table-cell; border: 1px solid black; vertical-align: middle;">
    This column should equal the height (no fixed-height allowed) of the 2 rows sitting to the right.
  </div>
  <div style="display: table-cell; width: 100px;">
    <div style="border: 1px solid black; margin-bottom: 2px;">
      Here is some sample text. And some additional sample text.
    </div>
    <div style="border: 1px solid black;">
      Here is some sample text. And some additional sample text.
    </div>
  </div>
</div>